ArkhamOverlay/Pages/LocalImages/LocalImagesViewModel.cs

```diff
@@ -0,0 +1,101 @@
+﻿using ArkhamOverlay.Data;
+using PageController;
+using System.Collections.Generic;
+using System.IO;
+using System.Windows;
+using System.Windows.Media;
+
+namespace ArkhamOverlay.Pages.LocalImages {
+    public class LocalImagesViewModel : ViewModel {
+        public LocalImagesViewModel() {
+            Packs = new List<LocalPack>();
+            CardTypes = new List<string> { "Asset", "Event", "Skill", "Scenario", "Agenda", "Act", "Enemy", "Treachery", "Location", "Investigator" };
```

Comment:
Minor: we seem to define this in a lot of places. I think we should consider consolidating these types and adding the overarching types (player, encounter, etc) as extensions.